### DISCUSSION TITLE
cpg0041-pccma

### DISCUSSION THREAD

**Discussion by ErinWeisbart** [broadinstitute/cellpainting-gallery#98] [Dataset descriptions]
Profile/data repo = [Link to profile/data repo]  
Publication/analysis repo = [Link to publication/analysis repo]  
Institution identifier = `alexslemonade`
Imaging Platform internal "champion" = `Erin`

**Workflow**
- [x] Maintainer adds dataset identifier to [prefixes](https://github.com/broadinstitute/cellpainting-gallery/blob/main/documentation/prefixes.md) document.
- [ ] Contributor has provided Per-Experiment metadata below (as described in [Cell Painting Gallery Metadata](https://broadinstitute.github.io/cellpainting-gallery-metadata/metadata_requirements/))
- [ ] Contributor has provided Per-Channel metadata below (as described in [Cell Painting Gallery Metadata](https://broadinstitute.github.io/cellpainting-gallery-metadata/metadata_requirements/))
- [x] Champion asks Erin/Shantanu to  1) open the prefix for transfer to `staging-cellpainting-gallery` and 2) create credentials that allow upload. Each credential is for a single prefix on the staging bucket; multiple people can assume that same credential.

**Transfer data to `staging-cellpainting-gallery`**
Instructions for using the staging bucket credentials are [here](https://broadinstitute.github.io/cellpainting-gallery/uploading_to_cpg.html).

REQUIRED:
- [ ] Image acquisition complete and uploaded (`/images`)
- [ ] Load_data complete and uploaded (`/workspace/load_data_csv`)
- [ ] Metadata complete and uploaded (`/workspace/metadata`)

OPTIONAL:
- [ ] Profiling complete and uploaded (`/workspace/profiles`)
- [ ] Feature extraction complete and uploaded (`/workspace/analysis`)
- [ ] Backend complete and uploaded (`/workspace/backend`)

**Project completion** 
These are the final steps to make a dataset fully public:
- [ ] Erin generates [harmonized metadata](https://broadinstitute.github.io/cellpainting-gallery-metadata/) and uploads to CPG
- [ ] Maintainer or champion updates cellpainting-gallery/README.md to add dataset to README.
- [ ] Maintainer transfers from `staging-cellpainting-gallery` to `cellpainting-gallery`.
- [ ] Erin/Shantanu disable staging prefix, delete data from staging, delete upload credentials.
- [ ] Maintainer or champion updates [Publications](https://github.com/broadinstitute/cellpainting-gallery/docs/publications.md) (In PR by data generator)

**Future supported/optional publication steps:**
- [ ] Run Distributed-BioFormats2Raw to create .ome.zarr files
- [ ] Upload (meta)data to IDR (images remain hosted in cellpainting-gallery) 
- [ ] Make IDR entry public
- [ ] Contributor adds the paper to https://github.com/cytodata/awesome-cytodata

**Comment by jenna-tomkinson** [comment]
@ErinWeisbart Thank you for getting this set up! We had one main question to ask:

_Can we upload plates of images to the Cell Painting Gallery (unpublished) as they're being collected before running CellProfiler pipelines?_ 

We would like to be able to access the images on our end in one place. Currently, we are using another method where we are sharing data so we would be uploading the data twice (once to share and second to AWS). If not, would this discussion only be for when we have fully analyzed all of the data that we collecting?

Thank you in advance for the clarification!

**Reply by ErinWeisbart** [reply]
You cannot upload directly to CPG.
However, you can upload directly to a staging bucket that, once reviewed by us for data structure compliance, will be transferred to the CPG. This could add up to a week of time in between your upload and full availability of the data on the CPG, though likely less time, especially if you let us know in advance when to expect data for review/transfer.

**Reply by gwaybio** [reply]
Thanks for this discussion @ErinWeisbart - we are getting closer to knowing when to expect data for review/transfer. This will likely happen at some point in the second half of 2025.

In the meantime, can you let us know if this process has changed? For example, in chatting with Jaime, there is now a metadata intake form I believe? Our aim is to be prepared when the time comes, so that we can save you (and ourselves) time on data transfer. 

One other question is if we can use google cloud as a staging bucket, or if it has to be AWS.

Thanks!

**Reply by ErinWeisbart** [reply]
Hey Greg. Yes, we have a per-batch metadata intake form (though, depending on experimental setup, one form can be applied to multiple batches. )

Staging must be our staging bucket on AWS. We will give you credentials that allow read/write to a single prefix (cpg0041-pccma) in the staging bucket. Either we can transfer images from staging to production batch by batch, when all images are collected, or at a specified point in time. Likewise for your `workspace` data products.

**Reply by gwaybio** [reply]
Thanks Erin! Ok, sounds good. 

Would it be possible for Jaime to send production ready images directly to you through existing Broad-internal channels? I will also connect with Jaime's data people if they can support this direct AWS transfer (I'd like to minimize transfers, if possible).

**Comment by gwaybio** [comment]
Update: about 1 year later, we have (nearly all) the data in hand! 🎉

@ErinWeisbart @jaimecheah is able to take on the intake process, replicating other datasets that she's sent over to cpg. Would it be ok if she gets started on this process? Thanks!

Tagging @jenna-tomkinson @jaceybronte and @jaclyn-taroni for visibility. Thanks all!

**Reply by ErinWeisbart** [reply]
Great! Some questions/comments:
- Is this all `images` or also a complete `workspace`? (if just images, when you do you anticipate workspace components being ready for deposition?)
- A metadata form needs to be filled out, per-batch if there are differences between batches, otherwise per-project is okay. Jamie is familiar with the form.
- @jaimiecheah, can you remind me if you are comfortable using AWS CLI to sync data to AWS? If so, I can send you credentials and explicit instructions. I believe we (imaging platform) have sync'd all other CDoT data that has gone to the CPG from Broad servers ourselves.

**Reply by gwaybio** [reply]
To answer point 1, it is just the images. Our plan is to release these publicly as soon as possible (no embargo). As for the workspace, we were hoping to go in a slightly different direction than the standard IP structure, and the timeline for upload there is currently unknown. A conservative estimate is 3 months.

**Reply by ErinWeisbart** [reply]
@gwaybio It's fine if the complete `workspace` isn't provided, but in order for us to put images in production we require `load_data_csv` and `metadata`.

**Reply by gwaybio** [reply]
Sounds good - this will be an easy lift. Tagging @jenna-tomkinson to take it from here! :)

**Reply by jenna-tomkinson** [reply]
@ErinWeisbart I am on it! Our estimated timeline to have profiles is the next 2 months. I will reach out when I have everything and confirm what to upload :)

**Reply by jaimecheah** [reply]
@ErinWeisbart - realizing I never responded to your previous question.
I have the images generated from all of our assay development and the Atlas. I think it's a question for @gwaybio and @jenna-tomkinson as to which specific images will need to go CPG. Once I know which ones will go, I can work on the metadata forms.
With regards to am I "comfortable using AWS CLI to sync data to AWS" - I have no idea what this means, so ummm .. no. LOL. Please send me explicit instructions in dummy mode so I can attempt this! Thank you!!

**Reply by ErinWeisbart** [reply]
@jaimecheah Do you have the images on a Broad server? If so, if you give me access I can sync from there.

**Reply by jaimecheah** [reply]
@ErinWeisbart - most of the images are here: //helium/cbnt_CDoTImaging/cdot_collab_xfer/Ready_to_transfer/Project Rambutan. I can figure out how to get you access, or I can transfer them to //helium/imaging_analysis if that's better. Let me check with Greg and Jenna as to exactly which ones should be going to CPG. Thanks!

**Reply by ErinWeisbart** [reply]
Generally the fewer transfers the better, so I'd prefer access in place if possible. A [Group Access Request](https://broad.service-now.com/sp?id=sc_cat_item&sys_id=92b804824fc302000629b3318110c7c6) to BITS should do it. (Broad internal link)

**Reply by jaimecheah** [reply]
@ErinWeisbart - submitted. Hopefully I did it correctly!

**Reply by ErinWeisbart** [reply]
@gwaybio @jenna-tomkinson I have access to the images on the server from CDoT (thanks @jaimecheah !).
Can one of you start a new comment thread below detailing what batches and plates should be transferred to CPG and if there is any renaming of folders that should be done?
 
The list of folders on the server are:
```
Atlas
CHP-134 REPO1 Screen
Round 2
Round 3
Round 4
SK-N-AS REPO1 Screen
```

**Comment by gwaybio** [comment]
Great! (sorry for my delayed reply)

The folders to upload are Round 2, Round 3, and Round 4 (@jaimecheah - do you need help finding Round 1?)

Please also upload CHP-134 (hold off on SK-N-AS for now). We are also replating Atlas, so please skip this too.

@jenna-tomkinson will let you know of folders beyond images, but we do not have those processed just yet.

**Comment by jaimecheah** [comment]
Re: Round 1 - I think Jenna has those, so we can re-upload them to the internal Broad server. The original images I believe are in deep archive by now.

Please hold off on CHP-134 for now. Jenna noted an issue with one of the folders so I need to re-upload and clean up that drive a little better. Will round back with that one has the green-light.

**Reply by jenna-tomkinson** [reply]
Hi @jaimecheah! Regarding the Round 1 data, I show on my local drive that it is labelled under the name `SN0313537` with the following folders:

```
.
├── 20240717_A-673_Re-imaged
├── 20240717_CHP-212_Re-imaged
├── 20240717_KP-N-YN_Re-imaged
├── 20240717_SK-N-MC_Re-imaged
├── 20240718_NB-1_Re-imaged
├── 20240718_SH-SY5Y_Re-imaged
├── 20240722_KNS-42_Re-imaged
├── 20240801_SK-N-AS_Re-imaged
├── BR00143976__2024-07-04T16_04_45-Measurement 2
├── BR00143977__2024-07-04T18_58_33-Measurement 1
├── BR00143978__2024-07-04T17_02_18-Measurement 1
├── BR00143979__2024-07-04T19_38_57-Measurement 1
├── BR00143980__2024-07-04T18_00_45-Measurement 1
├── BR00143981__2024-07-04T20_18_13-Measurement 1
└── README
```

Does this look accurate? If so, how would we go about re-uploading to the Broad server you mentioned above?

**Comment by jenna-tomkinson** [comment]
@ErinWeisbart Chatting with Greg currently, there was a minor mistake in his message. You are clear to upload the following images:

- Round 2 (pilot)
- Round 3 (pilot)
- Round 4 (pilot)
- SK-N-AS (drug screen)

Please hold off for the Atlas and CHP-134 folders and we are working on getting the Round 1 data uploaded.

**Reply by ErinWeisbart** [reply]
@jenna-tomkinson @gwaybio I see in your files that you have a 2 tiered batch.  e.g. Round2 AND Plate3_A673_Reimage for plates
```
BR00145438__2024-11-20T23_10_55-Measurement 3/	BR00145439__2024-11-20T23_25_43-Measurement 3/	BR00145440__2024-11-20T23_43_51-Measurement 3/
```

CPG only supports a single batch nesting. How would you like to convert your data?
Preserve only the `Round2` level?
Preserve only the `Plate3_A673_Reimage` level?
Concatenate them like `Round2 _Plate3_A673_Reimage` ?


Likewise for `SK-N-AS REPO1 Screen/` there is `Original REPO1 Screen/` and `REPO1 Row O Repeat/`.
How should those be handled?

**Reply by jenna-tomkinson** [reply]
Hi @ErinWeisbart,

Thank you for bringing this to our attention!

The Round 2 dataset, along with the other rounds and the SK-N-AS dataset, includes reimaging of plates as part of cell line optimization or issues during acquisition. At the time these datasets were processed, we structured the data to distinguish which images should be included, so our pipelines and load data CSVs do not currently reflect a single-batch nesting format.

We’re hoping to include this dataset in the Cell Painting Gallery for the grant and wanted to clarify the requirements. Is single-batch nesting strictly required, or are there cases where the current structure can be supported? If changes are needed, would updating the load data CSVs be sufficient, or would this require reprocessing the data?

Collecting profiles across both the pilot datasets and the “SK-N-AS” screens has been a substantial effort, so we’re hoping to minimize additional processing.

As we’re planning to recollect the Atlas and expand to additional drug screens, it would also be helpful to understand how reimaging or other technical issues during acquisition should be handled in the folder structure moving forward, particularly for the collection by @jaimecheah. This guidance would help ensure consistency for both CPG uploads and our internal pipelines.

Thank you in advance for your guidance!

**Reply by ErinWeisbart** [reply]
Yes, single batch nesting is strictly required. Updating load_data.csv's is fine - there is no need to reprocess data as the image data itself isn't changing, just changing file paths. As I mentioned above, you can keep all of information in your double-batch nesting, it will just need to be a name concatenation instead of a folder nesting (`Round2_Plate3_A673_Reimage` instead of `Round2/Plate3_A673_Reimage`.
Re-imaging is fine and putting a suffix into a batch or plate identifier is a fine way to denote it.
Our structure requirements are detailed in our documentation here https://broadinstitute.github.io/cellpainting-gallery/data_structure.html

**Reply by jenna-tomkinson** [reply]
Thanks for confirming!

Please go ahead and update the names with use the round as a prefix (e.g., `Round2_Plate3_A673_Reimage`, etc.).

As for the SK-N-AS repo1 screen, feel free to move the two folders in `REPO1 Row O Repeat` into the `Original REPO1 Screen` folder so there is no double-nesting. But, could you add "Row_O_repeat" as a prefix for those two folders like for the round as you described in your message? 

This means we will have 29 folders/plates for the screen. We will need to include that 27 of the folders will include images for row O that are not included during analysis/removed in load data CSV files, please let me know where that will be included for CPG.

Update from the last message, the images for the CHP-134 repo1 screen have been fixed! It will also have a same double-nested structure where three of the plates needed to be reuploaded and are added to a separate folder (double-nested). I have already started a pipeline with this structure, but please feel free to also do the same move as the SK-N-AS above but no need for a prefix.

**Reply by jenna-tomkinson** [reply]
I feel like it would be beneficial to visualize the expected structure in the context of the PCCMA work for the CPG.

@ErinWeisbart can you confirm if what I have is accurate?

```
cellpainting-gallery/
└── cpg0041-pccma
    └── broad
        └── images
        │   ├── <date>_PCCMA_pilot
        │   │   ├── illum
        │   │   │   ├── BR00145439
        │   │   │   │   ├── BR00145439_IllumDNA.npy
        │   │   │   │   └── BR00145439_IllumRNA.npy
        │   │   │   └── BR00145440
        │   │   └── images
        │   │       ├── Round2_Plate3_A673_Reimage
        │   │   │   │   ├──BR00145439
        │   │   │   │   ├──BR00145440
        │   └── <date>_PCCMA_SK-N-AS_repo1_screen
        │   │   └── images
        │   │       ├── Row_O_repeat_BR00149048
...
        └── workspace
```

**Reply by ErinWeisbart** [reply]
The source is `alexslemonade` not `broad`
You have added an additional nesting under `images` that should not be there.
The <date> isn't a strictly necessary component of a batch name.

If I were to follow the naming convention you currently have and concatenate the multi-batch it would look like this:

```
cellpainting-gallery/
└── cpg0041-pccma
    └── alexslemonade
        └── images
        │   ├── Round2_Plate3_A673_Reimage
        │   │   ├── illum
        │   │   │   ├── BR00145439
        │   │   │   │   ├── BR00145439_IllumDNA.npy
        │   │   │   │   └── BR00145439_IllumRNA.npy
        │   │   │   └── BR00145440
        │   │   └── images
        │   │   │   │   ├──BR00145439
        │   │   │   │   ├──BR00145440
        │   ├── Round2_Plate3_<othercellline>_Reimage
        │   ├── Round2_Plate3_<othercellline>_Reimage
        │   ├── SK-N-AS_REPO1_Screen_Original
        │   └── SK-N-AS_REPO1_Screen_RowO_Repeat
        │   │   └── images
        │   │       ├── Row_O_repeat_BR00149048
...
        └── workspace
```

**Reply by jenna-tomkinson** [reply]
Thank you clarifying this visual, it is very helpful for our group to get a good understanding of how this structure applies to the project.

We decided we want to remove the "Row_O_repeat" prefix for the plates in the SK-N-AS screen. We also added how we are expecting the CHP-134 data to be formatted (see below). Let me know if there is anything else we need to discuss regarding the `images` side! 🙂 

```
cellpainting-gallery/
└── cpg0041-pccma
    └── alexslemonade
        └── images
        │   ├── Round2_Plate3_A673_Reimage
        │   │   ├── illum
        │   │   │   ├── BR00145439
        │   │   │   │   ├── BR00145439_IllumDNA.npy
        │   │   │   │   └── BR00145439_IllumRNA.npy
        │   │   │   └── BR00145440
        │   │   └── images
        │   │   │   │   ├──BR00145439
        │   │   │   │   ├──BR00145440
        │   ├── Round2_Plate3_<othercellline>_Reimage
        │   ├── Round2_Plate3_<othercellline>_Reimage
        │   ├── SK-N-AS_REPO1_Screen_Original
        │   └── SK-N-AS_REPO1_Screen_RowO_Repeat
        │   │   └── images
        │   │       ├── BR00149048
        │   ├── CHP-134_REPO1_Screen
        │   └── CHP-134_Reupload
        │   │   └── images
        │   │       ├── BR00149048
...
        └── workspace
```

**Comment by jenna-tomkinson** [comment]
@ErinWeisbart Hi there! 👋 

The Way Lab and Jaime's team have been pushing through the work with the PCCMA project and wanted to reach back out to you with a couple of questions! We are hoping to add the computed profiles from the pilot data (Rounds 1-4) and want to work through any issues before uploading the repo1 screens and altas.

1. Have you uploaded the current pilot data to the "staging bucket" and performed the renaming as mentioned above? We have realized that we do not have visibility for this.
2. Is there a way that we can have visibility or a way to access the PCCMA staging bucket for CPG so we can make updates like adding profiles + illum files + loaddata CSVs? Or is this something that you, as our `Champion`, will only have access to and make the necessary changes?
3. Assuming it is only you who can make changes in the staging bucket for us, then how should I upload the pilot data (`workspace` and `illum`) to the shared bucket (CDoT from Jaime) to make this process easiest?

We appreciate your help as our Champion for this project!

**Reply by jenna-tomkinson** [reply]
Hi Erin,

I believe I have answered my questions above via the documentation here: https://broadinstitute.github.io/cellpainting-gallery/uploading_to_cpg.html

We are ready to upload data for the pilot dataset, which I assume will go under `workspace` per the [documentation](https://broadinstitute.github.io/cellpainting-gallery/data_structure.html). The next step for me would be to get the temporary AWS credentials. Please let me know how to access this, and I will upload the pilot data profiles and relevant metadata.

Thanks!

**Reply by ErinWeisbart** [reply]
Hi Jenna - 
`1)` I have not uploaded the data that Jaime moved to a Broad server to cellpainting-gallery-staging. As discussed in a previous thread, there is significant rearrangement of the data that needs to happen before it is structured properly for upload and therefore is beyond the scope of the data transfer that I originally offered. In order for me to spend the time organizing your data I would need to be compensated - our policy is [detailed here](https://broadinstitute.github.io/cellpainting-gallery/contributing_to_cpg.html#acknowledging-cpg-maintainers-role-in-deposition). I am still happy to sync from the server to staging once the data has been appropriately organized, or please let me know if you would like me to perform the data organization for compensation.

`2/3)` I will email you with credentials that will allow anyone with that set of credentials to read and write to your prefix in staging (i.e. you can share them between anyone in your team).

**Reply by jenna-tomkinson** [reply]
Hi Erin! @ErinWeisbart 

I have confirmed a successful upload of the pilot dataset (`images` and `workspace`) to the cpg staging bucket under `cpg0041-pccma`. When you have a chance, I would like to move forward to the next step to verify that my transfer complies with the data structure requirements. 

Thanks!

**Reply by ErinWeisbart** [reply]
I see in staging that you have uploaded 54 batches of images but your workspace folders are organized into 12 batches. There needs to be a 1:1 concordance between all batches/plates of images and metadata so you will need to make the workspace/metadata and workspace/load_data_csv for the other 40 batches

**Reply by jenna-tomkinson** [reply]
Hi Erin,

Thanks for bringing this to our attention! For reference, we did not process this pilot dataset in 54 batches (reimaged images are processed with the plate ID they came from). 

To align with this standard, is the expectation that we should copy the loaddata_csv file related to the specific plate ID for each batch of images (e.g., Round1/BR0014378__.../load_data.csv is the same loaddata csv as Round1_A-673_Reimage.../BR0014378__.../load_data.csv)? This would create the 1:1 concordance.

This would be the same question with `metadata`.

Given that we are now aware of the concordance, we believe that this should not be an issue for the other experiments, as they are being processed in a way that we can better accommodate this expectation.

**Reply by ErinWeisbart** [reply]
If they weren't processed as separate batches, I would suggest a different rearrangement of the data for clarity. You will still need to have a load_data.csv and metadata for each plate (and it's fine if they are duplicated, though if there is technical difference between the acquisitions I would expect that to be captured in the metadata (e.g. one was at 20X and one was at 40X then there would be a `Microscope_Objective_Magnification` column). This is our current [harmonized metadata ontology](https://github.com/broadinstitute/cellpainting-gallery-metadata/blob/main/harmonized_ontology.json) that we will need to be able to create for your data before it goes to production, so it may be worth looking through it now and seeing if there is anything in the ontology that is different between your reimages).

Because we try to keep "batch" as a functional unit, I suggest the following arrangement where the "<cellline>_Reimage" suffix is a part of the plate name for the illum images and your batches reflect actual functional batches. You don't need to rename the plates as scanned as the metadata appended to the plate name disambiguates them and the load_data maps them to the correct illum, though you are welcome to.

```
cellpainting-gallery/
└── cpg0041-pccma
    └── alexslemonade
        └── images
        │   ├── Round2_Plate3
        │   │   ├── illum
        │   │   │   ├── BR00145439_A673_Reimage
        │   │   │   │   ├── BR00145439_A673_Reimage_IllumDNA.npy
        │   │   │   │   └── BR00145439_A673_Reimage_IllumRNA.npy
        │   │   │   └── BR00145440_A673_Reimage
        │   │   └── images
        │   │   │  ├──BR00145439__2024-11-21T00_17_04-Measurement4
        │   │   │  ├──BR00145439__2024-11-21T03_52_34-Measurement8
        │   │   │  ├──BR00145440__2024-11-21T00_35_04-Measurement4
        │   │   │  ├──BR00145440__2024-11-21T04_10_33-Measurement8
...
```

Does this make sense?

**Reply by ErinWeisbart** [reply]
You _could_ add to or change the /images plate names if you want to, e.g.
`BR00145439__2024-11-21T00_17_04-Measurement4_A673_Reimage` OR
`BR00145439__A673_Reimage` OR
`BR00145439_A673_Reimage__2024-11-21T00_17_04-Measurement4`

**Reply by ErinWeisbart** [reply]
To be clear, you don't **have** to rearrange your data into a smaller number of batches that represent how they were actually processed. It's up to how you would like your data presented/organized. The only strict requirement is 1:1 load_data and metadata.

**Reply by jenna-tomkinson** [reply]
Thank you for this clarification! I believe it makes sense and am I working on the fix now, will ping you once it is ready.

**Reply by jenna-tomkinson** [reply]
@ErinWeisbart, thank you for your advice! I made the edits that should achieve 1:1 concordance based on the above, where I moved the images into the batches as you showed. Please let me know what else I need to fix after you give it a look!

**Reply by ErinWeisbart** [reply]
Can I confirm that I am understanding that: 
- the load_data.csv's reflect the images that were used for your analysis?
- for batches where a plate was imaged multiple times, the first acquisition captures the whole plate and then subsets of the plate were reimaged during subsequent acquisitions?

Can you confirm that there is no different acquisition metadata between re-imagings? (e.g. different objective, binning, channels, etc.)

**Reply by jenna-tomkinson** [reply]
> the load_data.csv's reflect the images that were used for your analysis?

Correct, the current load_data.csvs reflect how we performed the analysis. We ran each plate with the original paths (cell lines that did not need to be reimaged) and with the paths to the reimaged data for the specific cell lines that needed reimaging. This means that any failed cell lines in the original are updated to the reimaged paths instead of "wasting" computation time on poor-quality images.

> for batches where a plate was imaged multiple times, the first acquisition captures the whole plate and then subsets of the plate were reimaged during subsequent acquisitions?

This is likely a good question for @jaimecheah to confirm. From my understanding, this is correct. First, the whole plate is captured, then any cell lines on that plate that did not image correctly the first time were rerun in a subsequent acquisition. 

> Can you confirm that there is no different acquisition metadata between re-imagings? (e.g. different objective, binning, channels, etc.)

From the XML files alone, I do not see any differences in objective or binning. Channels will be the same across the entire PedCP90 (pilot) dataset: standard Cell Painting + one channel Brightfield. What I did find were very minor differences in exposure time and excitation power for some of the reimaged data. Given these are the two things I found different, what does CPG expect for this?

@jaimecheah should be able to confirm this if there was anything explicitly changed during acquisition that is not captured by the metadata.

**Comment by ErinWeisbart** [comment]
@jenna-tomkinson As an FYI, I have updated the checklist in the top comment to show our current workflow. Please note that the workflow now asks you to provide per-experiment metadata and per-channel metadata as a comment here. As you have many batches, these can be provided on a per-batch basis or a per-superset basis (i.e. let me know which batches they apply to). Please note that any per-channel or per-experiment value that varies from plate to plate should instead be included in the platemap.

This is a relatively new workflow for us so please let me know if you have any questions or if any of our documentation around the harmonized metadata can be improved.

**Reply by jenna-tomkinson** [reply]
Hi Erin, thanks for the heads up! Can you clarify how this should look in this discussion? Is this a CSV file that I need to add or just a copy-paste of the list? I understand exactly what is necessary from the documentation but would appreciate an example I can reference.

**Reply by ErinWeisbart** [reply]
A copy paste list is fine. Here is an example of another dataset providing the [information in thread](https://github.com/broadinstitute/cellpainting-gallery/discussions/180#discussioncomment-17723002).

**Reply by jenna-tomkinson** [reply]
Hi Erin,

Due to the large number of batches (12), I decided to create a markdown file with the per-experiment and per-channel info. There are only two things that I believe @jaimecheah should be able to confirm, which are the `CP_Version` and `Cell_Line_Modification`. Please let me know if the current format would be acceptable, thanks!

[cpg0041_pccma_pilot_data_per_experiment_channel.md](https://github.com/user-attachments/files/31800144/cpg0041_pccma_pilot_data_per_experiment_channel.md)

**Reply by ErinWeisbart** [reply]
Thanks Jenna. A few questions/comments about metadata and image file:
- You've listed `Microscope_Objective_NA` as 1.0 which I am suspicious of. @jaimecheah can you confirm the NA of 20X objective on the Revivify Opera Phenix that you used for these images?
- You have listed the cell line names as `Cell_Line_Type`, which I am happy to convert , but can you please provide the cell *type* for each of these cell *lines*? e.g. U2OS is the *name* of a line of *osteosarcoma* or *bone cancer* cells.
- Can you please provide the disambiguating metadata for when the same plate is acquired multiple times. This can be done as a comprehensive comment here if it is simply explainable by text or it can be included in the load_data.csv.

**Reply by jenna-tomkinson** [reply]
> You have listed the cell line names as Cell_Line_Type, which I am happy to convert , but can you please provide the cell type for each of these cell lines? e.g. U2OS is the name of a line of osteosarcoma or bone cancer cells.

I have updated the file to include the cell line type (e.g., pediatric cancer type) along with a number next to it if we have multiple cell lines from the same type. There is only one cell that I could not find a type for, which is noted in the file. @gwaybio @jaimecheah could either of you confirm what the CHLA-262 cancer type is, please?

> Can you please provide the disambiguating metadata for when the same plate is acquired multiple times. This can be done as a comprehensive comment here if it is simply explainable by text or it can be included in the load_data.csv.

We acquired the re-imaging data (parts of the same plate multiple times) because the original plate acquisition failed for specific cell lines for several reasons, mainly due to poor-quality images that would not yield accurate biological profiles. Given that we are using the plates in this dataset (PedCP90) to optimize experimental conditions, a confounding variable of low-quality images would impact our analysis and selection of density and time point per cell line for the Atlas or REPO1 screens.

Please let me know if this is sufficient text to explain the various imaging of the same plates in this dataset or if any specifics are needed to be included.

**Reply by jenna-tomkinson** [reply]
I have updated the unknown cell line per a chat with @gwaybio, but will need to be double-checked. Attached is the updated file:
[cpg0041_pccma_pilot_data_per_experiment_channel.md](https://github.com/user-attachments/files/32117707/cpg0041_pccma_pilot_data_per_experiment_channel.md)

**Reply by ErinWeisbart** [reply]
@jenna-tomkinson Can you confirm that you didn't perform any perturbations on these cell lines (i.e. no additional genetic or chemical perturbation)?
Can you tell me which pieces of metadata are variable between batches? I see that the cell lines are different between batches and I see that some are acquired in 2024 and some in 2025. Are there any other differences are all the rest of the per-experiment and per-channel metadata shared?

**Comment by jaimecheah** [comment]
Hi all,

Catching up - I think I caught all the questions that I need to resolve.
Please see below. Let me know if I missed anything!

Best,
Jaime


>> @jaimecheah <https://github.com/jaimecheah> can you confirm the NA of
20X objective on the Revivify Opera Phenix that you used for these images?

Confirming: Imaging was performed on a Revvity Opera Phenix in widefield
mode using a 20× water-immersion objective (NA 1.0).  Nine fields were
acquired per well using a 2160 × 2160-pixel camera with 2 × 2 binning,
producing 1080 × 1080-pixel images.

I have confirmed with Masha (for this and a variety of other projects) that
20X/NA 1.0 is the only option we have for a 20X water-immersion objective
on the Phenix that we used for this study.

 >> @jaimecheah <https://github.com/jaimecheah> Can you confirm that there
is no different acquisition metadata between re-imagings? (e.g. different
objective, binning, channels, etc.)

Confirming: The overall acquisition metadata won't have changed - 20X/NA
1.0 water-immersio objective, 2x2 binning, channels all remain the same.
The only things that will have changed are the individual channel setting
for plan, gain and power to enhance acquisition for various cell lines.

The way we typically acquired images for plates with multiple cell lines
was by using the U2OS cells on the plate as the anchor to set parameters
and imaged the entire plate. After reviewing the images, we went back and
set individual channel settings for each line to improve focus,
intensities, etc, if needed. Hopefully this helps!

>> @jaimecheah <https://github.com/jaimecheah> could either of you confirm
what the CHLA-262 cancer type is

Looking back at our files, it came in labeled on the vial as CHLA-262, but
it is listed on the COG website (where we procured it) as COG-R-262. Listed
as a rhabdomyosarcoma.

https://www.cccells.org/tables/Rhabdomyosarcoma.php

On Fri, Sep 11, 2026 at 12:18 PM Jenna Tomkinson ***@***.***>
wrote:

> I have updated the unknown cell line per a chat with @gwaybio
> <https://github.com/gwaybio>, but will need to be double-checked.
> Attached is the updated file:
> cpg0041_pccma_pilot_data_per_experiment_channel.md
> <https://github.com/user-attachments/files/32117707/cpg0041_pccma_pilot_data_per_experiment_channel.md>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/cellpainting-gallery/discussions/98?email_source=notifications&email_token=BKPPWLEVZEM6VGZDDGNSO7L5OQQUVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBUGAZTINJRUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-18403451>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKPPWLF32KL3TWMLQP5JU4T5OQQUVAVCNFSNUABHKJSXA33TNF2G64TZHM2DEMZYGEZDONZTHNCGS43DOVZXG2LPNY5TOMZSGM3DENNBOYBA>
> .
> Triage notifications, keep track of coding agent tasks and review pull
> requests on the go with GitHub Mobile for iOS
> <https://github.com/notifications/mobile/ios/BKPPWLBOIGD3HRZRHXMNFUL5OQQUVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBUGAZTINJRUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ>
> and Android
> <https://github.com/notifications/mobile/android/BKPPWLEVGZYG6TY4BYSFDXL5OQQUVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCOBUGAZTINJRUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>.
> Download it today!
> You are receiving this because you were mentioned.Message ID:
> <broadinstitute/cellpainting-gallery/repo-discussions/98/comments/18403451
> @github.com>
>


-- 

*Upcoming Out of Office:*

*September 17-18 (NCI visit, limited emails)*

*September 24 - October 4 (Berlin Marathon + OOO, no emails)*


--------------------------------------

Jaime H. Cheah, Ph.D.

Associate Director, Drug Discovery

Director of Scientific Outreach and Collaborative Screening

Center for the Development of Therapeutics

Broad Institute of MIT and Harvard

415 Main Street

Cambridge, MA 02142

***@***.***

Pronouns: she/her/hers


[image: BroadInstLogoforDigitalRGB.png] <https://www.broadinstitute.org/>
